packages/bpk-component-split-input/src/BpkSplitInput.module.scss

```diff
@@ -0,0 +1,36 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /* stylelint-disable */
+
+@import '~bpk-mixins/index';
+
+.BpkSplitInput {
+  display: flex;
+  width: 100%;
+  padding-top: $bpk-spacing-xs();
```

Comment:
same here, small typo

Reply:
Addressed in [c0d9dfe](https://github.com/Skyscanner/backpack/pull/2472/commits/c0d9dfe6662c1b56c6bd9b8864e36a93086a5924)